config/new_test.go

```diff
@@ -7,21 +7,24 @@ import (
 	"crypto/tls"
 	"testing"
 
+	"github.com/hashicorp/hcp-sdk-go/config/geography"
 	"github.com/hashicorp/hcp-sdk-go/profile"
 	requirepkg "github.com/stretchr/testify/require"
 )
 
 func TestNew_Default(t *testing.T) {
 	require := requirepkg.New(t)
 
+	conf := geography.NewConfigNorthAmerica()
```

Comment:
I am wondering if we need to change this to `configFromGeography`, which is a new function in the PR and makes call to geography.NewConfigNorthAmerica().

Reply:
Definitely, I will add further tests which excersize the new functionality. This was kinda a POC branch until the RFC got approved